bika/lims/content/sampletype.py

```diff
@@ -198,6 +254,158 @@ def ContainerTypesVocabulary(self):
+        the stickers selected in admitted.
+
+        :return: A DisplayList
+        """
+        admitted = self.getAdmittedStickers()
+        if not admitted:
+            return DisplayList()
+        voc = DisplayList()
+        stickers = getStickerTemplates()
+        for sticker in stickers:
+            if sticker.get('id') in admitted:
+                voc.add(sticker.get('id'), sticker.get('title'))
+        if voc.index > 0:
+            return voc
+        return DisplayList()
+
+    def _large_default_voc(self):
+        """
+        Returns the vocabulary to be used in
+        AdmittedStickerTemplates.large_default
+
+        If the object has saved not AdmittedStickerTemplates.admitted stickers,
+        this method will return an empty DisplayList. Otherwise it returns
+        the stickers selected in admitted.
+
+        :return: A DisplayList
+        """
+        admitted = self.getAdmittedStickers()
+        if not admitted:
+            return DisplayList()
+        voc = DisplayList()
+        stickers = getStickerTemplates()
+        for sticker in stickers:
+            if sticker.get('id') in admitted:
+                voc.add(sticker.get('id'), sticker.get('title'))
+        if voc.index > 0:
+            return voc
+        return DisplayList()
+
+    def setDefaultSmallSticker(self, value):
+        """
+        Sets the small sticker ID defined as default.
+
+        :param value: A sticker ID
+        """
+        if value is None:
+            logger.error(
+                "Setting wrong 'AdmittedStickerTemplates/small_default' value"
+                " to Sample Type '{}'"
+                .format(self.getId()))
+            return
+        if not isinstance(value, str):
+            logger.error(
+                "Setting wrong 'AdmittedStickerTemplates/small_default' value"
+                " type to Sample Type '{}'"
+                .format(self.getId()))
+            return
+        field = self.getField('AdmittedStickerTemplates')
+        new_value = field.get(self)
+        new_value[0]['small_default'] = value
+        field.set(self, new_value)
+
+    def setDefaultLargeSticker(self, value):
+        """
+        Sets the large sticker ID defined as default.
+
+        :param value: A sticker ID
+        """
+        if value is None:
+            logger.error(
+                "Setting wrong 'AdmittedStickerTemplates/large_default' value"
+                " to Sample Type '{}'"
+                .format(self.getId()))
+            return
+        if not isinstance(value, str):
+            logger.error(
+                "Setting wrong 'AdmittedStickerTemplates/large_default' value"
+                " type to Sample Type '{}'"
+                .format(self.getId()))
+            return
+        field = self.getField('AdmittedStickerTemplates')
+        new_value = field.get(self)
+        new_value[0]['large_default'] = value
+        field.set(self, new_value)
```

Comment:
Again, the code of `setDefaultLargeSticker` is almost exact to `setDefaultSmallSticker`, please use a single function e.g. `setDefaultSticker(self, value, SIZE=STICKER_SMALL)`